apps/client/src/app/components/not-found/not-found.component.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
Please remove DOCTYPE, head and body tags. We only need the specific contents of the component.